<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="header" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QLineEdit" name="headerTop">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>ACU ready for combat.</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QLineEdit" name="headerBottom">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Log yourself in, commander.</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="_splashSide" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="splash">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="extraOptionsToggle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Login parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="extraOptionsFrame">
           <property name="hidden">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="extraOptionsLayout" columnstretch="2,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="serverPortField">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>Server port</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="replayServerPortField">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>Replay server port</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLineEdit" name="apiURLField">
              <property name="placeholderText">
               <string>API URL</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QComboBox" name="environmentBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="replayServerHostField">
              <property name="placeholderText">
               <string>Replay server host</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="serverHostField">
              <property name="placeholderText">
               <string>Server host</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="ircServerHostField">
              <property name="placeholderText">
               <string>IRC server host</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="ircServerPortField">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>IRC server port</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rememberCheckbox">
           <property name="text">
            <string>Remember me</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newAccountButton">
           <property name="text">
            <string>Create new Account</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renameAccountButton">
           <property name="text">
            <string>Rename your Account</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="steamLinkButton">
           <property name="text">
            <string>Link your account to Steam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forgotPasswordButton">
           <property name="text">
            <string>Forgot Login or Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bugreportButton">
           <property name="text">
            <string>Report a Bug</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stayOfflineButton">
           <property name="text">
            <string>Stay Offline</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="quitLoginWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>Continue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loginButton</tabstop>
  <tabstop>extraOptionsToggle</tabstop>
  <tabstop>serverHostField</tabstop>
  <tabstop>serverPortField</tabstop>
  <tabstop>replayServerHostField</tabstop>
  <tabstop>replayServerPortField</tabstop>
  <tabstop>ircServerHostField</tabstop>
  <tabstop>ircServerPortField</tabstop>
  <tabstop>apiURLField</tabstop>
  <tabstop>environmentBox</tabstop>
  <tabstop>rememberCheckbox</tabstop>
  <tabstop>newAccountButton</tabstop>
  <tabstop>renameAccountButton</tabstop>
  <tabstop>steamLinkButton</tabstop>
  <tabstop>forgotPasswordButton</tabstop>
  <tabstop>bugreportButton</tabstop>
  <tabstop>stayOfflineButton</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_accepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extraOptionsToggle</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_toggle_extra_options()</slot>
  </connection>
  <connection>
   <sender>environmentBox</sender>
   <signal>currentIndexChanged()</signal>
   <receiver>Dialog</receiver>
   <slot>on_fill_extra_options()</slot>
  </connection>
  <connection>
   <sender>steamLinkButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_steamlink_account()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bugreportButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_bugreport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_request_quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forgotPasswordButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_forgot_password()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameAccountButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_rename_account()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newAccountButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>on_new_account()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rememberCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>on_remember_checked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stayOfflineButton</sender>
   <signal>released()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_accepted()</slot>
  <slot>on_new_account()</slot>
  <slot>on_rename_account()</slot>
  <slot>on_steamlink_account()</slot>
  <slot>on_forgot_password()</slot>
  <slot>on_bugreport()</slot>
  <slot>on_remember_checked(bool)</slot>
  <slot>on_request_quit()</slot>
 </slots>
</ui>
